### ISSUE TITLE
exemptions scraper should run in smart update mode where it updates until no more updates

### ISSUE THREAD

**Issue by OriHoch** [OpenBudget/budgetkey-data-pipelines#34]
(relevant after #13 is merged)

### reproduction steps
* run the exemptions pipeline
* it runs the following for each publisher:
  * download 1 page of latest data each time
  * data is updated in dump.to_sql

#### expected
* once newest data is inserted - stop and continue to next publisher
* this might be supported using frictionlessdata/datapackage-pipelines#64
* but, maybe there is a different way

#### actual
* currently we limit to number of pages (at the moment it's 5 pages = 50 items)
* we insert / update all 50 items, then move to next publisher
* if a publisher has more then 50 new items, not all items will be inserted
* if publisher has only 1 new entry - it will still update all 50 entries

**Comment by OriHoch** [comment]
@akariv commented:
> A more robust way would be to check if the data for this document already exists, and if not then do the scraping. This will allow in the future more correct behaviours, such as periodically re-scraping records that are still "in progress" (i.e data might still be changing) and not scraping records that have been concluded.
Checking if a document exists could be done e.g. by hitting the DB and collecting the list of existing publications per publisher id (iirc it's not such a big list).

**Comment by OriHoch** [comment]
fixed in #13 - it now fetches all IDs from DB and compares with IDs from the scraper - so it only inserts new items

opened a new issue to handle the update of items - #38
